### PR TITLE
Minor A11y updates, post axe audit

### PR DESCRIPTION
**Pull request by jaffrepaul**

### CODE REVIEW

File: apps/migrator-e2e/src/integration/migrator.spec.ts [hunk 1]
```diff
@@ -11,6 +11,7 @@ const clearProtractor = (): Cypress.Chainable<JQuery<HTMLElement>> => {
 
 const enterProtractor = (value: string): void => {
   cy.get('textArea').first().type(value, { parseSpecialCharSequences: false })
+  cy.get('textArea').first().should('exist')
```
Comment (admah) [inline apps/migrator-e2e/src/integration/migrator.spec.ts:14]: I don't think this is necessary based on the line above.

File: apps/migrator-e2e/src/integration/migrator.spec.ts [hunk 2]
```diff
@@ -138,4 +140,9 @@ describe('Migrator app', () => {
       migrations.forEach((migration: IMigration) => verifyMigration(migration))
     })
   })
+
+  it('Has no detectable a11y violations on load', () => {
+    // Test the page at initial load
+    cy.checkA11y()
```
Comment (admah) [inline apps/migrator-e2e/src/integration/migrator.spec.ts:146]: Are there ways to output what the failures are from here? I see in the logs that there are 2 failing items, but not what they are.

File: apps/migrator/components/migratorEditor.tsx [hunk 3]
```diff
@@ -121,10 +121,10 @@ const MigrateEditor = (): ReactElement => {
       </div>
       <div className="flex flex-wrap-reverse justify-center md:justify-between items-center p-4 md:py-0 gap-y-4">
         <div className="text-center md:text-left">
-          <h4 className="font-bold">Want to dig deeper?</h4>
+          <h2 className="font-bold">Want to dig deeper?</h2>
```
Comment (admah) [inline apps/migrator/components/migratorEditor.tsx:124]: Is the `h2` too big for this space?

**Review by admah — CHANGES_REQUESTED** [review]
I had some questions. I also see some failing checks.

File: apps/migrator-e2e/src/integration/migrator.spec.ts (hunk 1 above)
Comment (jaffrepaul) [reply apps/migrator-e2e/src/integration/migrator.spec.ts:14]: Oops. Forgot to set this PR to draft. I'm poking around here to figure out why the test got flaky recently.

File: apps/migrator/components/migratorEditor.tsx (hunk 3 above)
Comment (jaffrepaul) [reply apps/migrator/components/migratorEditor.tsx:124]: The h2 update was by way of the a11y check which spotted that the markup had "h" tags but not in sequential order, which will confuse screen readers. The tag isn't used for font size here, just more logical flow down the tree.

File: apps/migrator-e2e/src/integration/migrator.spec.ts (hunk 2 above)
Comment (jaffrepaul) [reply apps/migrator-e2e/src/integration/migrator.spec.ts]: The plugin outputs more detail on each caught issue in the console with links to a11y references. We can look into customizing what gets displayed in the command log.

**Review by admah — APPROVED** [review]
Looks great! Thanks for taking care of the accessibility issues!

**Pull request merged**